docs/project/2024/iulia_mirela.buta/index.md

```diff
@@ -0,0 +1,130 @@
+# Fire alarm system
+A fire alarm system trigerred by a fire source or other sort of light source in the proximity
+
+:::info
+
+*Author*: Buta Iulia-Mirela \
+*GitHub Project Link*: https://github.com/UPB-FILS-MA/project-ButaIulia
+
+:::
+
+
+
+## Description:
+
+A fire alarm system trigerred by a fire source or other sort of light source in the proximity. It includes the following features:
+- Detection of the fire using a Flame Sensor.
+- Activating a buzzer alarm when fire is detected.
+- Displaying a warning message on a LCD display.
+
+## Motivation
+For my first embedded project I wanted to do something usefull for the everyday life and also to practice all the skills that I gained at the laboratory and at the course, and so I choose to  develop a reliable and cost-effective fire alarm system using Raspberry Pi Pico that can detect and alert users in case of fire emergencies.
+
+## Arhitecture
+The architecture of the project is the following:
+1	Raspberry Pi Pico W: Microcontroller unit responsible for processing and controlling the system.
+2. ST7789 Display: Display module for visual notifications and system status.
+3. Fire Sensor: Detects the presence of fire or smoke.
+4. Buzzer: Audible alarm for alerting users.
+5. Power Supply: Provides power to the Raspberry Pi Pico and peripherals.
+
+## Log
+
+<!-- write every week your progress here -->
+
+### Week 6 - 12 May
+
+### Week 7 - 19 May
+
+### Week 20 - 26 May
+
+## Hardware:
+
+1. Raspberry Pi Pico Board – 1
+2. 16×2 I2C LCD Display – 1
+3. Flame Sensor – 1
+4. Breadboard – 1
+5. RGB LED Module – 1
+6. Jumper Wires – 10
+7. Micro-USB Cable – 1
+9. Buzzer - 1
+
+## Connection Overview:
+* The fire sensor is connected to one of the GPIO pins of the Raspberry Pi Pico for detecting fire or smoke.
+* The buzzer is connected to another GPIO pin to sound the alarm when triggered.
+* The ST7789 display is connected to the Raspberry Pi Pico via SPI for displaying system status and alerts.
+* Power supply is connected to Raspberry Pi Pico for powering the entire system.
+
+### Schematics
+
+-+-------------------------+
+- |      Fire Sensor        |
+- +-------------------------+
+-
+-                 |
+-                 | (Signal)
+-                 V
+-         +-----------------+
+-         | Raspberry Pi Pico|
+-         +-----------------+
+-                 |
+-      +----------+----------+
+-      |                     |
+-      V                     V
+- +---------+           +-----------+
+- |  Buzzer |           |  ST7789   |
+- +---------+           +-----------+
++```
+++-------------------------+
++ |      Fire Sensor        |
++ +-------------------------+
++
++                 |
++                 | (Signal)
++                 V
++         +-----------------+
++         | Raspberry Pi Pico|
++         +-----------------+
++                 |
++      +----------+----------+
```

Comment:
This is not correct and does not render, this is the git diff that the website shows. You can clock click *Commit Changes* and it makes the modifications. Please make the ascii chart enclosed in ```.